colorcoder/ColorCoderReportUtil.java

```diff
@@ -0,0 +1,45 @@
+package colorcoder;
+

```

Comment:
How does class name ColorCoderReportUtil relates to its encapsulated functionalities ?